.eslintrc.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
 		'computed-property-spacing': [ 1, 'always' ],
 		'func-call-spacing': 1,
 		indent: [ 1, 'tab', { SwitchCase: 1 } ],
+		'jsx-a11y/accessible-emoji': 1,
+		'jsx-a11y/click-events-have-key-events': 1,
+		'jsx-a11y/heading-has-content': 1,
+		'jsx-a11y/href-no-hash': 1,
+		'jsx-a11y/html-has-lang': 1,
+		'jsx-a11y/iframe-has-title': 1,
+		'jsx-a11y/img-has-alt': 1,
+		'jsx-a11y/label-has-for': 1,
```

Comment:
>unless you think that would just confuse people?

Taking my own confusion as an example, I'd say yes 😄 

Your plan sounds reasonable, though I expect it to be a source of confusion for any project using the rules in the meantime.